Library/ConfigParser.cpp

```diff
@@ -264,6 +264,13 @@ bool ConfigParser::GetSectionVariable(std::wstring& strVariable, std::wstring& s
 			}
 			else
 			{
+				
+				const std::wstring optionValue = m_Skin->GetParser().ReadString(strVariable.c_str(), selector.c_str(), L"");

```

Comment:
I'm not sure what you would gain from making this change. Why or where would you want to read an option from a meter/measure? (This is where a comment describing your change would be helpful in the PR description!)